tasks/build.yml

```diff
@@ -0,0 +1,97 @@
+---
+
+- name: Create prom2teams directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ sansible_prom2teams.user }}"
+    group: "{{ sansible_prom2teams.group }}"
+    mode: 0750
+  with_items:
+    - "{{ sansible_prom2teams.path.install }}"
+    - "{{ sansible_prom2teams.path.install }}/bin"
+    - "{{ sansible_prom2teams.path.install }}/prom2teams"
+    - "{{ sansible_prom2teams.path.log }}"
+    - "{{ sansible_prom2teams.path.pid }}"
+
+- name: Copy wrapper shell script
+  become: yes
+  copy:
+    dest: "{{ sansible_prom2teams.path.install }}/bin/prometheus_wrapper"
+    group: "{{ sansible_prom2teams.group }}"
+    mode: 0755
+    owner: "{{ sansible_prom2teams.user }}"
+    src: prometheus_wrapper
+
+- name: Add Deadsnakes Repo
```

Comment:
Might be worth adding a note in the README about why we are forcing a newer version of Python3.

Reply:
Done.